charts/steward/values.yaml

```diff
@@ -0,0 +1,76 @@
+# Default values for steward.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+runController:
+  image:
+    repository: stewardci/stewardci-run-controller
+    tag: 191217_cc244c0
+    pullPolicy: IfNotPresent
+  resources:
+    limits:
+      cpu: 100m
+      memory: 64Mi
+    requests:
+      cpu: 50m
+  podSecurityContext: {}
```

Comment:
are we running as root?

Reply:
Currently the controller run as root. But we can change this (different PR).